### DISCUSSION TITLE
How do I change the station ID?

### DISCUSSION THREAD

**Discussion by abenedet78** [briis/hass-weatherflow#51] [Q&A]
Thanks for the development of this. It's been working great. 
I need to change the station ID. How can this be done for an existing installation?

Thank you

**Comment by briis** [comment]
Unfortunately that is not possible as the ID number forms part of the unique key for each sensor. 
However you can delete the current installation if the ID no longer exists and then add the integration again, with your new station id and you should be fine - but you will lose any customisations you have done to the sensors. If the old station is still in production you can have them both loaded at the same time.
